graylog2-server/src/main/java/org/graylog/events/processor/aggregation/AggregationEventProcessorConfig.java

```diff
@@ -194,37 +203,40 @@ public ValidationResult validate() {
         if (!series().isEmpty() && isConditionsEmpty()) {
             validationResult.addError(FIELD_CONDITIONS, "Aggregation with series must also contain conditions");
         }
+        if (eventLimit() <= 0) {
+            validationResult.addError(FIELD_EVENT_LIMIT, "Event limit must be greater than 0.");
+        }
```

Comment:
Maybe it would make sense to also set some kind of upper boundary. At least it should be Integer.MAX_VALUE

Reply:
It is not possible to check for Integer.MAX_VALUE here, because an exception is thrown beforehand. The error message then looks like this:
![Screenshot 2023-08-08 at 17 37 42](https://github.com/Graylog2/graylog2-server/assets/25727579/545226cf-0d73-42cb-ad79-16919f380701)

Reply:
@mpfz0r @EvaZg Should we set a maximum limit value that is smaller than Integer.MAX_VALUE?

Reply:
I have added a configurable maximum limit for events

Reply:
@moesterheld please re-review, or remove the review fail :)

Reply:
@mpfz0r only added these as comments but will add my approval as they have been addressed.